server/src/main/java/io/druid/segment/loading/OmniSegmentLoader.java

```diff
@@ -52,12 +54,14 @@
   public OmniSegmentLoader(
```

Comment:
Right now one of the functions of omni-segment-loader is to have an explicit point where the dependencies must be materialized. There is a place where the LoadSpec's method is called, and that is the point where the dependency extensions must be loaded and available.